packages/ui/src/footer/top-section.tsx

```diff
@@ -60,6 +74,7 @@ const Circle: FC<CircleProps> = ({ children, href, variant }) => {
       href={href}
       rel="noreferrer"
       target="_blank"
+      aria-label={`Got to ${name}`}
```

Comment:
This is a link, so the "Go to..." is extraneous.

```suggestion
      aria-label={name}
```